cumulusci/cli/cci.py

```diff
@@ -1053,6 +1053,41 @@ def org_list(runtime, plain):
     persistent_table.echo(plain)
 
 
+@org.command(name="prune", help="Removes all expired orgs from the current project")
+@pass_runtime(require_project=False, require_keychain=True)
+def org_prune(runtime):
+    org_configs = {
+        org: runtime.keychain.get_org(org) for org in runtime.keychain.list_orgs()
+    }
+    scratch_configs = getattr(runtime.project_config, "orgs__scratch", {})
+
+    expired_orgs_removed = []
+    org_shapes_skipped = []
+    active_orgs_skipped = []
+    for org, org_config in org_configs.items():
```

Comment:
I think you could do:

```suggestion
    for org, org_config in runtime.keychain.orgs.items():
```

and skip setting org_configs above.

Also if it were me I'd call it `org_name` instead of `org`